air/src/stack/op_flags/tests.rs

```diff
@@ -0,0 +1,143 @@
+use super::{
+    generate_evaluation_frame, OpFlags, NUM_DEGREE_4_OPS, NUM_DEGREE_6_OPS, NUM_DEGREE_7_OPS,
+};
+use vm_core::{Felt, ONE, ZERO};
+
+// CONSTANTS
+// ==================================================================================================
+
+pub const DEGREE_7_OPCODE_STARTS: usize = 0;
+pub const DEGREE_7_OPCODE_ENDS: usize = DEGREE_7_OPCODE_STARTS + 63;
+pub const DEGREE_6_OPCODE_STARTS: usize = DEGREE_7_OPCODE_ENDS + 1;
+pub const DEGREE_6_OPCODE_ENDS: usize = DEGREE_6_OPCODE_STARTS + 31;
+pub const DEGREE_4_OPCODE_STARTS: usize = DEGREE_6_OPCODE_ENDS + 1;
+pub const DEGREE_4_OPCODE_ENDS: usize = DEGREE_4_OPCODE_STARTS + 31;
+
+#[test]
+fn op_flags() {
```

Comment:
Now that I'm seeing how it looks, I'd probably just have 3 separate tests: e.g., `degree_7_op_flags`, `degree_6_op_flags` etc.

And then, within each tests I'd have the corresponding loop, and maybe get rid of the `assert...` helper functions (they are pretty small and unique for each test).